src/operations/command.ts

```diff
@@ -37,7 +37,10 @@ export interface CommandOperationOptions extends OperationOptions, WriteConcernO
 
 /** @internal */
 export interface OperationParent {
-  s: { namespace: MongoDBNamespace };
+  s: {
+    namespace: MongoDBNamespace;
+    options?: BSONSerializeOptions;
```

Comment:
I was thinking that `bsonOptions` would be another inheritable property like `readConcern`/`writeConcern`/`readPreference` etc. outside of the `s`. Should we at least consider renaming this to `bsonOptions`?